internal/cli/opsmanager/backup/restores_start.go

```diff
@@ -33,6 +33,7 @@ const (
 	automatedRestore = "AUTOMATED_RESTORE"
 	httpRestore      = "HTTP"
 	onlyFor          = "'%s' can only be used with %s"
+	createTemplate   = "Created restore job.\n"
```

Comment:
[q] can we return the ID here? this is useful as users may need to describe it later to know the status

Reply:
Added it now -- i was a bit confused at first because there seems to be multiple jobs returned but I wasn't sure how that came about.

Reply:
Oh... that may be true, in a sense, this was a weird thing we found out and is not properly documented (we filled a ticket for it) the create job returns and array of 1, always, I think this is something like, `Results[0].ID` should be fine (I'm not sure if with templates you can directly access the first element)

Reply:
yeah none of the api examples returned multiple results